### PR TITLE
Merge RSG-12(DAL) into develop

### PR DESCRIPTION
**Pull request by askliar**
DAL and IDAL are finished. Kindly ask you to review

### CODE REVIEW

File: src/WebUI/Startup.cs [hunk 1]
```diff
@@ -52,6 +52,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc();
 
+            //Add DAL
+            services.AddScoped<IDAL, DAL>();
```
Comment (askliar) [inline src/WebUI/Startup.cs:56]: @yevgen-kabanets What does this fragment does? Is it inner ASP.NET IOC?

File: src/DAL/DAL.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.Data.Entity;
+
+namespace DAL
+{
+    public class DAL : IDAL
+    {
+        private const string COACH_ROLE = "Coach";
+        private const string PARTICIPANT_ROLE = "Participant";
```
Comment (askliar) [inline src/DAL/DAL.cs:15]: @yevgen-kabanets You should definitely try to avoid CAPSLOCK CASE!

Comment (GasperPaul) [inline src/DAL/DAL.cs:15]: Java is strong with this one.

File: src/WebUI/Startup.cs (hunk 1 above)
Comment (yehorchankov) [inline src/WebUI/Startup.cs:56]: @emohat1r Yes, it is.
[Here](https://codewala.net/2015/04/30/your-dependency-injection-ready-asp-net-asp-net-5/) is some more info.
P.S. it seems like ninject is not needed

**Comment by GasperPaul** [comment]
This is already in #5

**Pull request closed without merging**